src/commands/tenmans.ts

```diff
@@ -23,6 +23,8 @@ import {
 let tenmansQueue: Member[] = [];
```

Comment:
I am...unsure about having mixed purpose for this variable. It's the 10mans queue but also the voting queue, which makes sense from the perspective of voting queues turning into proper queues, but the naming is making the code below hard to understand.

Reply:
I recognize that it's a bit weird. With support for multiple queues up next on the list though, I think that a lot of the voting/queue starting code is going to need significant refactor anyways, which makes me hesistant to add something cleaner here that will probably get thrown out when all of the considerations from that are known.

Reply:
We can do it then, just wanted to make sure it was being considered for that future change